### PR TITLE
thermald: use named thermal zones

### PR DESCRIPTION
**Pull request by geohot**
This fixes the CPU having the wrong zones. It also changes the zones used by the GPU from the "lowf" to the "usr" ones.

### CODE REVIEW

**Comment by geohot** [comment]
CI issues?

File: selfdrive/thermald/thermald.py [hunk 1]
```diff
@@ -50,10 +50,25 @@
...
+    if not n.startswith("thermal_zone"):
+      continue
+    with open(os.path.join("/sys/devices/virtual/thermal", n, "type")) as f:
+      tz_by_type[f.read().strip()] = int(n.lstrip("thermal_zone"))
+
 def read_tz(x):
   if x is None:
     return 0
 
+  if isinstance(x, str):
```
Comment (adeebshihadeh) [inline selfdrive/thermald/thermald.py:67]: think it's cleaner to do this in `HARDWARE.get_thermal_config`, then it always returns a path you can read from

Comment (geohot) [reply selfdrive/thermald/thermald.py:67]: I disagree with the way it's currently structured now. The numbers are meaningless and should imo be handled at the last possible moment.

Comment (geohot) [reply selfdrive/thermald/thermald.py:67]: That's probably better

**Comment by geohot** [comment]
Good to merge?

**Comment by geohot** [comment]
Good to merge?

File: selfdrive/thermald/thermald.py (hunk 1 above)
Comment (pd0wm) [reply selfdrive/thermald/thermald.py:67]: Ok, to merge like this. I can do the refactor when I'm back from holiday.

**Pull request merged**